<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolBarClipWidget</class>
 <widget class="QWidget" name="ToolBarClipWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>170</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>170</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="cutButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../잘라내기.png</normaloff>../잘라내기.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="copyButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../복사.png</normaloff>../복사.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pasteButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../붙여넣기.png</normaloff>../붙여넣기.png</iconset>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
